### ISSUE TITLE
Water box level is always at 0%  (Roborock T7 / roborock.vacuum.a11)

### ISSUE THREAD

**Issue by timothy-morph** [homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum#259]
I got a Roborock T7 (roborock.vacuum.a11)
Everything is working fine except the water box level is always at 0% in homebridge and home app
But the log in the homebridge is: 
`INF getWaterSpeed | roborock.vacuum.a11 | WaterBoxMode is 202 over miIO. Converting to HomeKit`

Is this normal? or is there anything I can do to make it right?
Thank you so much 👍

**Comment by afharo** [comment]
Hi @timothy-morph!

I think it's related to the model not being in the list of [models/models.js](https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/blob/master/models/models.js).

Can you try in your installation by editing the `./models/models.js` and adding the snippet below right in the previous line to `"roborock.vacuum.a10"`?

```
  "roborock.vacuum.a11": [
    {
      speed: speedmodes["gen4+custom"],
      waterspeed: watermodes["gen1+custom"],
    },
  ],
```

If it works for you, we can add it in the next release 👍

**Comment by timothy-morph** [comment]
Hi @afharo , after adding the model the water box level works perfectly !
Thanks for the quick help! 👍 😃

a little bit off topic here, Is it normal that when I start the cleaning(no matter if it's room cleaning or not), all the room cleaning switches will all switch on at the same time?

Example: 
If I start the living room cleaning, it does it perfectly and go back, but it also turn on all other room cleaning switches while it's cleaning.

**Comment by afharo** [comment]
> Hi @afharo , after adding the model the water box level works perfectly !
> Thanks for the quick help! 👍 😃

@timothy-morph thank you for confirming! I've published a new version `0.13.2` with those changes then!

> a little bit off topic here, Is it normal that when I start the cleaning(no matter if it's room cleaning or not), all the room cleaning switches will all switch on at the same time?
> 
> Example:
> If I start the living room cleaning, it does it perfectly and go back, but it also turn on all other room cleaning switches while it's cleaning.

Personally, I don't use the room features, but after taking a quick look at the code, it looks like that's the case. When the Home app asks for the state of any Room Switch, we currently only check if the vacuum is cleaning (in general).

This might be fixed when we implement the discussion started in https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/issues/132#issuecomment-632716279 (and follow up comments): Basically, the user will switch ON the rooms to be cleaned and, then, trigger the clean by starting the cleaning. If no rooms are switched ON, we'll trigger a general cleaning (or something along those lines :) )

**Comment by bao3** [comment]
My vacuum is ` Roborock T7 Pro (roborock.vacuum.a9)`
I use the plugin version `0.13.2` ,and Enable the “waterbotx(beta)” , but I still got the error: 
`INF getWaterSpeed | roborock.vacuum.a11 | WaterBoxMode is 202 over miIO. Converting to HomeKit`

**Comment by timothy-morph** [comment]
If your model is roborock.vacuum.a9, then I guess you'll have to add this new model into the following file [models/models.js](https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/blob/master/models/models.js) before the a10 model

```
"roborock.vacuum.a9": [
  {
    speed: speedmodes["gen4+custom"],
    waterspeed: watermodes["gen1+custom"],
  },
],
```

**Comment by afharo** [comment]
@timothy-morph Thank you! Published as `0.13.3`
